src/isar_turtlebot/robotinterface.py

```diff
@@ -49,15 +50,17 @@ def __init__(self):
         )
         self.current_task: Optional[str] = None
         self.inspection_status: Optional[TurtlebotStatus] = None
+        self.id: str = ""
 
-    def schedule_task(self, task: Task) -> Tuple[bool, Optional[Any], Optional[Joints]]:
-        run_id: str = self._publish_task(task=task)
-        return True, run_id, None
+    def schedule_task(self, task: Task) -> Tuple[bool, Optional[Joints]]:
+        self.id = self._get_turtlebot_id()
+        self._publish_task(task=task)
+        return True, None
 
     def mission_scheduled(self) -> bool:
         return False
 
-    def mission_status(self, mission_id: Any) -> MissionStatus:
+    def mission_status(self, mission_id: UUID) -> MissionStatus:
```

Comment:
```suggestion
    def mission_status(self) -> MissionStatus:
```

Implement mission_id as optional input in ISAR if it is necessary for some robots?

Reply:
Good idea :+1: